ios/RNIterableAPI/ReactIterableAPI.swift

```diff
@@ -97,6 +97,13 @@ class ReactIterableAPI: RCTEventEmitter {
         
         IterableAPI.email = email
     }
+
+    @objc(setEmail:withAuthToken:)
+    func set(email: String?, authToken: String?) {
+        ITBInfo()
```

Comment:
Its good swift does not log the email and authToken. In future, Android can have this approach as well 👍🏼

Reply:
Ah yeah, all we log is the function call